ynr/apps/data_exports/views.py

```diff
@@ -79,11 +87,20 @@ def get_context_data(self, **kwargs):
         )
         page = paginator.get_page(self.request.GET.get("page", "1"))
         context["page_obj"] = page
+
         return context
 
 
 class DataExportView(DataFilterMixin, View):
     def get(self, request, *args, **kwargs):
+        if "download" not in self.request.GET and (
+            request.user.is_authenticated
+            and not CSVDownloadReason.objects.filter(user=request.user).exists()
+        ):
```

Comment:
This code will never show the survey form to an unauthenticated user because this branch will never be followed if `request.user.is_authenticated` is false. Was that the intent? Assuming it wasn't, its not quite as simple as just removing this check because if you do, it throws

```
TypeError: Field 'id' expected a number but got <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7dc5de165490>>
```

Can I leave this one with you to follow up?

Reply:
Moot since I reverted this